main.py

```diff
@@ -0,0 +1,44 @@
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+import pyspark.sql.functions as F
+import argparse
+from cli.variant_disease_gene import variant_disease_gene
+#from cli.variant_disease_gene import variant_disease
+
+parser = argparse.ArgumentParser(
+                    prog='main',
+                    description="Open Targets Genetics annotation")
+parser.add_argument('--app_name', type=str, default="OTG Annotation", help='name to give to the spark application')
+parser.add_argument('--spark_mem', type=str, default="12g", help='Amount of total memory to give to spark')
+parser.add_argument('--spark_cpu', type=str, default="10", help='Total number of cpu to use')
+parser.add_argument('--variants_query', type=str, default=None, help='Path of a txt file storing the variants to investigate in the format chr_pos_ref_alt.')
+parser.add_argument('--gnomad_af', type=str, default="gnomad_nfe", help='Population for which the allele frequency is to be retrieved. Check the README for a list of available populations.')
+parser.add_argument('--tag', type=bool, default=False, help='Flag indicating whether to match both for lead and tag variant. Default is false meaning only the lead variants will be considered.')
+parser.add_argument('--out', type=str, default="v2dg_out", help='The name of the file where to store the results of the query.')
+args = vars(parser.parse_args())
+
+def main(app_name: str , spark_mem: str , spark_cpu: str, variants_query: str,out: str, gnomad_af: str, tag: bool):
+    conf = (
+        SparkConf()
+        .set("spark.driver.memory", spark_mem)
+    )
+    spark_session = SparkSession\
+        .builder\
+        .master(f"local[*]")\
+        .appName("test") \
+        .config("spark.local.dir", "/scratch/bruno.ariano/tmp_pyspark") \
```

Comment:
Adjust the code here so that temp folder points to the user scratch space. Add a to make the tmp folder if missing because I think spark will not do this for you automatically

Reply:
Changed with the env variable for the user